components/pricing_modal/content.tsx

```diff
@@ -58,11 +61,24 @@ function Content(props: ContentProps) {
     const product = useSelector(selectSubscriptionProduct);
     const products = useSelector(selectCloudProducts);
 
+    const findProductBySku = (sku: string) => {
+        return Object.values(products || {}).find(((product) => {
+            return product.sku === sku;
+        }));
+    };
+
+    const findProductByID = (id: string) => {
+        return Object.values(products || {}).find(((product) => {
+            return product.id === id;
+        }));
+    };
```

Comment:
We may want to move these to a more central util area, or add them as specific selectors that can be called with useSelector so we can re-use them later. You can even incorporate the RecurringInterval as mentioned below, which will abstract that away from the developer for the most part.